Project/Controllers/ToDosController.cs

```diff
@@ -18,19 +19,19 @@ public class ToDosController : ControllerBase
         [HttpPost]
         [ProducesDefaultResponseType]
         [ProducesResponseType(StatusCodes.Status201Created)]
-        public ActionResult<ToDos> PostToDo(ToDos todo)
+        public ActionResult<ToDos> PostToDo(ToDos todos)
```

Comment:
`todos` naming is confusing, because is a plural but the object is not actually an array.

Reply:
This refers to both the argument name and the class name. You are getting here a single ToDo item, not an array/list. The class name should be singular, since it represents 1 item.